packages/deployer/src/deploySolutionItems.ts

```diff
@@ -234,13 +234,21 @@ export function deploySolutionItems(
       itemId: string,
       status: common.EItemProgressStatus,
       costUsed: number,
-      createdItemId: string // supplied when status is EItemProgressStatus.Created
+      createdItemId: string // supplied when status is EItemProgressStatus.Created or .Finished
     ) => {
       // ---------------------------------------------------------------------------------------------------------------
       percentDone += progressPercentStep * costUsed;
       /* istanbul ignore else */
       if (options.progressCallback) {
-        options.progressCallback(percentDone, options.jobId);
+        if (status === common.EItemProgressStatus.Finished) {
+          const event = {
+            event: common.SItemProgressStatus[status],
+            data: createdItemId
```

Comment:
On this line I believe I want the "itemId" not the "createdItemId". I want the ID of the template Item that the createdItem was created from.

Reply:
sorry--misread spec. thanks for catch